### PR TITLE
feat: Handle deletion with mentions (WEBAPP-5463)

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

**Comment by gregor** [comment]
Please tag with ticket number. I suppose this is related to WEBAPP-5463.

File: app/script/view_model/content/InputBarViewModel.js [hunk 1]
```diff
@@ -398,15 +400,40 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
...
     }
   }
 
+  updateMentions(data, event) {
+    const textarea = event.target;
+    const value = textarea.value;
+    const previousValue = this.input();
+    const mentions = this.parseForMentions(previousValue, this.conversationEntity().participating_user_ets());
+    const lengthDifference = value.length - previousValue.length;
+    this.logger.log(this.updateMentionRanges(mentions, this.selectionStart(), this.selectionEnd(), lengthDifference));
```
Comment (ffflorian) [inline app/script/view_model/content/InputBarViewModel.js:419]: Can we add some helpful text to the log? Like "Remaining mentions: ..."?

Comment (atomrc) [reply app/script/view_model/content/InputBarViewModel.js:419]: This is debug code that will be removed in the very near future ;)

Comment (AndyLnd) [reply app/script/view_model/content/InputBarViewModel.js:419]: This line is just there to consume it for now. It will be gone in the next PR.

**Pull request merged**